.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - dev
+      - main
+    tags:
+  pull_request:
+  release:
+
+jobs:
+  check-copyright:
+    name: Copyright
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: License header
+        run: .scripts/check-copyright-notice.sh
```

Comment:
[golangci-lint](https://golangci-lint.run/usage/linters/#goheader) can also do this for you.

Reply:
yes, the great linter config change is due with https://github.com/hyperledger-labs/go-perun/issues/132

Reply:
Our header check just became much easier because we don't contain year numbers (`20XX`) any more. So most of the logic in the script `check-copyright-notice.sh` is probably obsolete and we might as well just use `goheader` and remove the script.

Reply:
The copyright check update is already part of https://github.com/hyperledger-labs/go-perun/issues/132.